CONTRIBUTING.md

```diff
@@ -26,9 +38,12 @@ VDK follows the [Code of Conduct](https://github.com/vmware/versatile-data-kit/b
 # Contributor License Agreement CLA.
 If you wish to contribute code, you must sign the [contributor license agreement](https://cla.vmware.com/cla/1/preview) (CLA). For any questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq) page.
 
-# Open Development
+# Pull Requests
 All contributors follow GitHub’s pull request (PR) workflow to submit changes. Before merging, all pull requests require two approvals.
 
+# Naming Your Branch
+Our branches follow the naming convention: `person/<username>/<branch-name>`
```

Comment:
This made me think, if we only ever label branches based on usernames and not tickets, we can probably discuss dropping the 'person' prefix from the convention. I'll bring this up on the daily.

Reply:
We discussed this during the daily and decided that what we have now is probably fine, and if anyone needs to violate the convention they are probably doing so for good reason.